zerver/lib/markdown/__init__.py

```diff
@@ -1218,6 +1245,42 @@ def find_proper_insertion_index(
             if uncle_link not in parent_links:
                 return insertion_index
 
+    def is_video(self, url: str) -> bool:
+        url_type = mimetypes.guess_type(url)[0]
+        if url_type is not None and url_type.startswith("video"):
+            return True
+        return False
+
+    def add_video(
+        self,
+        root: Element,
+        url: str,
+        class_attr: str = "message_inline_video",
```

Comment:
What's the thinking behind selecting `message_inline_video` rather than `message_inline_image` with perhaps some other CSS classes? 

We will need to coordinate this decision with mobile in the "api design" stream; perhaps you can start a topic there about this detail?

Reply:
I think it is just cleaner to add a separate class for it than mixing it with `message_inline_image`.